src/test/javascript/spec/component/lecture/lecture-import.component.spec.ts

```diff
@@ -0,0 +1,124 @@
+import { ComponentFixture, fakeAsync, TestBed, tick } from '@angular/core/testing';
+import { FormsModule } from '@angular/forms';
+import { NgbHighlight, NgbPagination } from '@ng-bootstrap/ng-bootstrap';
+import { ButtonComponent } from 'app/shared/components/button.component';
+import { SortService } from 'app/shared/service/sort.service';
+import { PageableSearch, SearchResult, SortingOrder } from 'app/shared/table/pageable-table';
+import { MockComponent, MockDirective, MockPipe } from 'ng-mocks';
```

Comment:
unused import (code warning)
```suggestion
import { MockComponent, MockDirective } from 'ng-mocks';
```

Reply:
Great to see my IDE doing its job… 🤖